acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,818 @@
+package gov.di_ipv_drivingpermit.pages;
+
+import gov.di_ipv_drivingpermit.utilities.Driver;
+import org.junit.Assert;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.logging.Logger;
+
+public class WelshLangDrivingLicencePageObject extends DrivingLicencePageObject {
+
+    private static final Logger LOGGER =
+            Logger.getLogger(WelshLangDrivingLicencePageObject.class.getName());
+
+    @FindBy(xpath = "//*[@id=\"main-content\"]/p/a/button")
```

Comment:
These are already in the object your extending this is still duplicated if the object or field or method is in the parent its not needed in the child